tests/test_size_change_monitor.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from backup.size_change_monitor import allowed_size_change, relative_size_change
+
+
+def test_allowed_size_change_for_small_files():
+    """ Returns large percentage for small files """
+    assert allowed_size_change(1) > 10000
+    assert allowed_size_change(100) > 2500
```

Comment:
Also test it is smaller than 10000? These tests currently don't say that much imo?
Or use an exact equality test? If people update the formula, tests should fail.

Reply:
Same for test below

Reply:
I just wanted to to test that it goes from really large to really small without hardcoding actual values, because they can change based on the variables (and probably will with some tweaking). If you have a better idea, let me know, otherwise I don't mind too much for just this backup check :)

Reply:
Yeah do an explicit equality check, imo it's ok if the tests fail because of a new formula.. it's different behavior.

Reply:
If it's not too much of an effort then make it explicit indeed. I don't expect we will change this often, and when we do then we just need to change the tests also. 👍